sympy/core/power.py

```diff
@@ -124,6 +124,16 @@ class Pow(Expr):
     | (-oo)**oo    | nan     |                                               |
     | (-oo)**-oo   |         |                                               |
     +--------------+---------+-----------------------------------------------+
+    | z = a+b*I    |         |                                               |
```

Comment:
I have left space between `a`, `+`, `b`, `I` intentionally, this looks better compared to with `space`.